src/js/addons/widgets/DropDownMenu.js

```diff
@@ -2,49 +2,52 @@ import Widget from "./Widget.js"
 
 import { getAddressSceneContent } from '../../core/codemirror/yaml-tangram.js';
 
-export default class DropDownMenu extends Widget {
-    constructor (datum) {
-        super(datum);
+let cm;
```

Comment:
I wanted to get rid of this one but there's still some functionality in this widget that refers to the editor, so this was not meant to be permanent.